awsproviderlint/passes/AWSAT004/README.md

```diff
@@ -0,0 +1,68 @@
+# AWSAT004
+
+The AWSAT004 analyzer reports hardcoded state hashes. Hash values were used
```

Comment:
Similar updates here to include `TestCheckResourceAttr()` and callout `TypeSet` as well. 👍